docs/secrets.mdx

```diff
@@ -0,0 +1,632 @@
+The Secret service provides applications with the ability to manage and access secrets such as database passwords,
+collection URLs and encryption keys. Secrets are generally used to access information in other systems.
+
+By using the Secret service you can avoid embedding secrets in configuration files or application code.
+
+The Nitric Secret service is provides a common programming interface across key Cloud secrets services including
```

Comment:
This line says `service is provides...` the extra `is` can be removed.

Reply:
Also, the phrasing 'across _key_ Cloud secrets services...' could be a little confusing. I'd suggest removing the word 'key' from the sentence since it's so overloaded in the secrets space.